implementations.tex

```diff
@@ -62,6 +62,9 @@ \section{Execution Based} \label{execution_based}
 to 0.
 
 To resume execution, the debug module sets a flag which causes the hart to execute a {\tt dret}.
+{\tt dret} is an instruction that only has meaning while in Debug Mode and
+not executing from the Program Buffer.
+Its recommended encoding is 0x7b200073.
```

Comment:
Are we really changing dret from a specified encoding to a recommended encoding?